axelor-contract/src/main/java/com/axelor/apps/contract/service/utils/FormulaProcessingResults.java

```diff
@@ -0,0 +1,76 @@
+package com.axelor.apps.contract.service.utils;
+
+import java.math.BigDecimal;
+
+public class FormulaProcessingResults {
+  private final BigDecimal ind1i;
+  private final BigDecimal ind2i;
+  private final BigDecimal ind1f;
+  private final BigDecimal ind2f;
+  private final BigDecimal ind1nm1;
+  private final BigDecimal ind2nm1;
+  private final BigDecimal initialUnitPrice;
+  private final BigDecimal price;
+
+  public FormulaProcessingResults(
```

Comment:
I'll let you check if we can use java reflection API to get dynamically the field name instead of defining constants